gensim/corpora/sharded_corpus.py

```diff
@@ -264,7 +264,7 @@ def init_shards(self, output_prefix, corpus, shardsize=4096, dtype=_default_dtyp
                 logger.info('Deriving dataset dimension from corpus: '
                              '{0}'.format(proposed_dim))
             else:
-                logger.warn('Dataset dimension derived from input corpus diffe'
+                logger.warning('Dataset dimension derived from input corpus diffe'
```

Comment:
Sure! I'll replace vertical indent with hanging indent for all the warnings here.